src/App.js

```diff
@@ -37,18 +36,20 @@ const App = () => {
     <Router>
       <ToastContainer />
       <Fragment>
-      <div className="flex justify-center align-middle">
-          <img src={require('./assets/novelology_newlogo.png')} alt="Novelology Logo" style={{ height: '12em', width: '12em' }} />
-        </div>
-            {/* <LoggedOutNavBar /> */}
-          <Navbar />
+
+        <NavigationBar />
```

Comment:
I put the NavigationBar separately in each pages because I thought sign in, register, and onboarding pages shouldn't display this navigationBar. What do you think?

Reply:
Should we not do it this way?

Reply:
Up to you i think! we can change it later too